### DISCUSSION TITLE
Allow Disabling Notifications for Required Reviewers on Workflows

### DISCUSSION THREAD

**Discussion by FlorianWendelborn** [community/community#18598] [Actions]
Currently, being listed as a Required Reviewer for an Environment/Workflow leads to a lot of useless Notifications from Github.

Just because a user is able to approve a deployment, doesn’t mean they want to be informed about every single deployment that takes places anywhere.

Therefore, it would be great to:

1. Allow users to opt-out of those notifications
  1.1. Optional: Support both globally and per action/environment
2. Optional: Allow organizations to make these notifications opt-in instead (as to not waste precious time of potentially hundreds of people with spam notifications)

**Comment by MathiasReker** [comment]
I think you can already configure this in the settings of the organisation repo.

**Reply by FlorianWendelborn** [reply]
If it is, I certainly can't find it. I'd expect it to be in the same place where the environment is configured in the first place but there's only an option for who is able to approve, not who should be notified
